submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionOnBehalfTest.kt

```diff
@@ -181,4 +181,48 @@ class SubmissionOnBehalfTest(
             assertThat(testFile).exists()
             assertThat(testFile).hasContent("submitter data")
         }
+
+    @Test
+    fun `14-5 On behalf with manager with another user submission update user Owner`() =
```

Comment:
@jhoanmanuelms this is the expected behavoir according to ugis so I added test to garantee we do not break this